### ISSUE TITLE
Cannot use list function with .query()

### ISSUE THREAD

**Issue by devvithelopper** [pouchdb/pouchdb#7460]
### Issue
I am trying to query a view in combination with a list function. Pouchdb would have to generate a url of the form

http://localhost:5984/mydatabase/_design/mydesigndoc/_list/mylist/myview

which at least the query() method basically does not seem to support.

I tried all of the following

`db.query('mydesigndoc/myview/mylist', {params...});`  
which pouchdb translates to the following base url
`http://localhost:5984/mydatabase/_design/mydesigndoc/_view/myview`

`db.query('mydesigndoc/mylist/myview', {params...});`
which pouchdb translates to the following base url
`http://localhost:5984/mydatabase/_design/mydesigndoc/_view/mylist`

and even
`db.query('_design/mydesigndoc/_list/mylist/myview', {params...});`
which pouchdb translates to
`http://localhost:5984/proverb/_design/_design/_view/mydesigndoc`

The last one might be interesting since one coult change the query method so that if the index name starts with `_` then use it as is...

Basically I don't see a way to query a view with a list function from pouchdb version 7.0.0

### Info
- Environment:  browser
- Platform: FF
- Adapter: ?
- Server: CouchDB

### Reproduce
Issues that others can reproduce are far easier to fix, we have a website that may help you provide a
live example of the bug @ https://pouchdb-bug-helper.glitch.me/

**Comment by willholley** [comment]
there is a plugin for list [here](https://python-pouchdb.marten-de-vries.nl/plugins.html). It supports both local list and remote list functions (http adapter), though may need updating to work with PouchDB 7.0
